rust/src/transformations/make_stable_expr/expr_alias/test.rs

```diff
@@ -0,0 +1,67 @@
+use polars::{df, lazy::frame::IntoLazy};
+use polars_plan::dsl::{all, col};
+
+use crate::{
+    domains::{AtomDomain, LazyFrameDomain, SeriesDomain},
+    metrics::SymmetricDistance,
+};
+
+use super::*;
+
+#[test]
+fn test_behavior() -> Fallible<()> {
```

Comment:
Unless I'm missing something, this isn't a test of our code, it's just making sure Polars works as expected? Maybe explain the motivation... are we not confident about the behavior of polars? Or is it a reminder of all the ways aliases could be used?

If it's worth keeping, maybe reorganize these two tests, so that each unit test will just do one operation with Polars, and then the same thing with our extension. Then, if we do realize there's an edge case we forgot, the addition will be one new unit that checks Polars out-of-the-box, and then checks our code.

Sorry if I'm reading this wrong.

Reply:
Yes, the aliasing transformation needs to change the output domain in the same way that polars changes dataframes when aliasing.